### DISCUSSION TITLE
Better indication of private browsing window

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#843] [Ideas]
### What feature would you like to see?

It's hard to recognize whether it's private mode or not without an indicator.
![image](https://github.com/user-attachments/assets/f694806b-2258-4fb0-ba06-8c8976c15cea)

Most browsers with private browsing/incognito mode including Firefox show indicator icon on titlebar/toolbar or different colors on their private window, and I'd appreciate to see it also on Zen to find out which window is in private mode or not easily.
![image](https://github.com/user-attachments/assets/c4e6dff6-578c-489d-8c79-7b8da4190b6a)
![image](https://github.com/user-attachments/assets/da54b318-a604-4a7e-a28b-2973a4690d51)
![image](https://github.com/user-attachments/assets/52393baa-33c9-4872-80a6-7f31c83cb754)

### What platform are you seeing the problem on?

macOS

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/416_

**Comment by Pat-JK** [comment]
Coming from Vivaldi, it would be nice to have a similar implementation as there. You can select any of your installed themes to be the theme specifically for private windows. Even just selecting a different highlight colour of the default theme would be helpful.

**Comment by artchiee** [comment]
This is a Feature I would like to see by default as well! the only way to verify your on private is by opening new tab , or by getting a theme suggested in the closed thread related.

**Comment by KodeBarista** [comment]
Adding a anonymous icon at the top of tab-bar would make it clear that current window is a private window.
Currently it's empty for private window & it shows the icon of current workspace when using in normal mode

![screenshot-2024-11-19-13-09-36](https://github.com/user-attachments/assets/60b40ede-65b9-4e3a-bcfc-096b309306a2)
